auto-update/Control/Reaper.hs

```diff
@@ -156,7 +156,7 @@ add settings@ReaperSettings{..} stateRef tidRef item =
   where
     cons NoReaper      = let !wl = reaperCons item reaperEmpty
                          in (Workload wl, spawn settings stateRef tidRef)
-    cons (Workload wl) = let wl' = reaperCons item wl
+    cons (Workload wl) = let !wl' = reaperCons item wl
```

Comment:
This is a good catch, thanks, and I'm a bit embarrassed for missing it. Question: to hopefully solve this in a more guaranteed way for the future, could you tell me if making the field of the `Workload` constructor a strict field also solves your problem?

Reply:
New commit with a strict data constructor instead of the bangs.